### PR TITLE
Remove the faucet redirect from JSON RPC

### LINKED ISSUE #1830: SDK: remove requestAirdrop from the JSON RPC API
The JSON `requestAirdrop` API is really just a thin wrapper around a drone, and makes unfortunate assumptions about where the drone is running and it's port.  With #1754 and the potential for many drones running around, we should fully decouple the drone from the RPC API and make the users fully aware of the drone they decide to use.  

Note that the drone itself may choose to implement something that looks very much like the existing JSON `requestAirdrop` API as a convenience for web-based users.

### PR DESCRIPTION
**Pull request by garious**
Attempt 2 https://github.com/solana-labs/solana/pull/3403

More of a reminder to delete the dependencies from example web-wallet and tic-tac-toe.  @mvines, anything change there?

Fixes #1830

### CODE REVIEW

File: client/src/rpc_request.rs [hunk 1]
```diff
@@ -38,7 +38,6 @@ pub enum RpcRequest {
     GetVersion,
     GetVoteAccounts,
     RegisterNode,
-    RequestAirdrop,
```
Comment (garious) [inline client/src/rpc_request.rs:41]: Do we need to keep a placeholder here for the enum value or does everyone specify the method with a string?

Comment (mvines) [reply client/src/rpc_request.rs:41]: It's all string based so this won't harm anything

**Comment by mvines** [comment]
https://github.com/solana-labs/solana/pull/3403#issuecomment-475102947 is still a problem for devnet though.  We do need some mechanism to distribute devnet tokens.   Or perhaps we wait a little longer when we can kill devnet entirely and have people develop directly on mainnet-beta

**Comment by garious** [comment]
@mvines, can the devnet validator embed the drone and expose it via a port instead through this JSON RPC API?

**Comment by mvines** [comment]
Oh right, my bad.  `solana airdrop` talks directly to the drone port so we're good there.   It's just the examples that need updating

**Comment by mvines** [comment]
Very stale

**Pull request closed without merging**